index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+    <title>Portfolio Setup</title>
+</head>
+<body>
+    <header>
+        <p>John Doe</p>
+        <button class="header-btn"><i class="fas fa-bars"></i></button>
+    </header>
+
+    <div class="container">
+        <ul class="headline">
+            <li class="primary-text">
+                <h1>Hey There. <br> I’m Alex</h1>
+            </li>
+            <li class="supporting-text">
+                <h2>I’am a Software Developer</h2>
```

Comment:
The paragraph (e.g., "I'm a software ...") is a `p` tag. Consider replacing it with a `p tag`

* This way will be semantically correct and the screen readers and other technologies will thank you

Reply:
[OPTIONAL] I think @ABDULALI3468 is confused with other templates. On this template, this area is the subtitle and you were right, it needs to be h2 element. 👍